de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataPresenterImpl.java

```diff
@@ -117,10 +123,16 @@ private void updateMetadataFromTemplateView() {
 
     @Inject
     public MetadataPresenterImpl(final MetadataView view,
```

Comment:
There are a small number of tests for this class that will need to be updated with the changes to the constructor.